Corzbank.Services/DepositService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using AutoMapper;
+using Corzbank.Data.Entities;
+using Corzbank.Data.Entities.Models;
+using Corzbank.Data.Enums;
+using Corzbank.Helpers;
+using Corzbank.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Corzbank.Services
+{
+    public class DepositService : IDepositService
+    {
+        private readonly GenericService<Deposit> _genericService;
+        private readonly IMapper _mapper;
+
+        public DepositService(GenericService<Deposit> genericService, IMapper mapper)
+        {
+            _genericService = genericService;
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<Deposit>> GetDeposits()
+        {
+            return await _genericService.GetRange();
```

Comment:
Easier to read if u first get a result, and then return it. Sometimes methods can be so complicated. Refactor in all places